doc/base/classes.xml

```diff
@@ -2967,20 +2969,28 @@
 	</methods>
 	<members>
 		<member name="animation" type="String" setter="set_animation" getter="get_animation" brief="">
+			The current animation's name.
 		</member>
 		<member name="centered" type="bool" setter="set_centered" getter="is_centered" brief="">
+			If true, offset is relative to the sprite's center. Top-left corner otherwise.
```

Comment:
Add default value.  Second sentence should be

Otherwise, offset is relative to the top-left corner.